src/codec/error.rs

```diff
@@ -6,10 +6,7 @@ use std::{error, fmt, io};
 #[derive(Debug)]
 pub enum RecvError {
     Connection(Reason),
-    Stream {
-        id: StreamId,
-        reason: Reason,
-    },
+    Stream { id: StreamId, reason: Reason },
```

Comment:
In cases like this, having the fields on separate lines means better diffs/blames...

Reply:
if we really cared about having better diffs/blames, we'd use comma-first style... 😜

/me is forcibly removed from the repo contributors

Reply:
I feel comma-first became a thing because of languages that don't allow trailing commas. Rust doesn't suffer from this fault. 😁

Reply:
this is [controlled by a length](https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#struct_variant_width).  Is there consensus about this _always_ being split?

Reply:
@seanmonstar trailing commas are ugly

Reply:
(comma first style is somehow not ugly because I'm Smart and Always Right)

Reply:
@olix0r In type definitions for structs, I like always breaking it up across lines.

Reply:
changing

```diff
-struct_lit_multiline_style = "PreferSingle"
+struct_lit_multiline_style = "ForceMulti"
```
pick your poison:

```diff
diff --git a/src/client.rs b/src/client.rs
index ed7b343..7bfab0e 100644
--- a/src/client.rs
+++ b/src/client.rs
@@ -79,7 +79,9 @@ where
             match codec.start_send(settings.into()) {
                 Ok(AsyncSink::Ready) => {
                     let connection = Connection::new(codec);
-                    Ok(Client { connection })
+                    Ok(Client {
+                           connection,
+                       })
                 }
                 Ok(_) => unreachable!(),
                 Err(e) => Err(::Error::from(e)),
@@ -91,7 +93,9 @@ where
             .map_err(::Error::from as fn(IoError) -> ::Error)
             .and_then(bind);
 
-        Handshake { inner: handshake }
+        Handshake {
+            inner: handshake,
+        }
     }
 
     /// Returns `Ready` when the connection can initialize a new HTTP 2.0
@@ -109,7 +113,11 @@ where
         self.connection
             .send_request(request, end_of_stream)
             .map_err(Into::into)
-            .map(|stream| Stream { inner: stream })
+            .map(|stream| {
+                Stream {
+                    inner: stream,
+                }
+            })
     }
 }
 
diff --git a/src/codec/error.rs b/src/codec/error.rs
index 1713e65..33c753f 100644
--- a/src/codec/error.rs
+++ b/src/codec/error.rs
@@ -53,7 +53,9 @@ impl error::Error for RecvError {
 
         match *self {
             Connection(ref reason) => reason.description(),
-            Stream { ref reason, .. } => reason.description(),
+            Stream {
+                ref reason, ..
+            } => reason.description(),
             Io(ref e) => e.description(),
         }
     }
diff --git a/src/codec/mod.rs b/src/codec/mod.rs
index 8d81150..01db79b 100644
--- a/src/codec/mod.rs
+++ b/src/codec/mod.rs
@@ -50,7 +50,9 @@ where
 
         let inner = FramedRead::new(delimited);
 
-        Codec { inner }
+        Codec {
+            inner,
+        }
     }
 }
 ```